extensions/wikia/EditPageLayout/EditPageLayoutHelper.class.php

```diff
@@ -195,8 +195,8 @@ static public function isCodeSyntaxHighlightingEnabled( Title $articleTitle ) {
 	static public function isInfoboxTemplate( Title $title ) {
 		$namespace = $title->getNamespace();
 
-		if ( $namespace === NS_TEMPLATE && class_exists( 'TemplateClassificationController' ) ) {
-			$tc = new TemplateClassificationController( $title );
+		if ( $namespace === NS_TEMPLATE && class_exists( 'TemplateClassification' ) ) {
```

Comment:
I think that since `TemplateClassification` is included in the `DefaultSettings.php` we can skip this check safely. :)